content/docs/latest/spire-helm-charts-hardened-about/namespaces.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Namespaces
+short: Namespaces
+description: Which namespaces to use to install the SPIRE Helm charts
+kind: spire-helm-charts-hardened-about
+weight: 210
+aliases:
+    - /docs/latest/helm-charts-hardened/namespaces
+---
+
+## 3 Namespace Configuration
+
+This is the recommended configuration, it automatically creates and deploys SPIRE across 3 namespaces.
+
+your-values.yaml snippet:
+```yaml
+global:
+  spire:
+    recommendations:
+      enabled: true
+    namespaces:
+      create: true
+```
+
+This will create `spire-server` and `spire system` namespaces, label them
+for proper Kubernetes Pod Security Standards operations, and deploy the various
+services appropriately.
+
+A third namespace is needed to house the Kubernetes Custom Resource Definitions
+as provided by the [spire-crds](https://artifacthub.io/packages/helm/spiffe/spire-crds) chart, as well as any Release information helm
+needs to store to manage the installation. This management namespace is specified
+directly to helm when installing/upgrading the release.
+
+## 2 Namespace Configuration
+
+It's not possible to have the SPIRE helm chart create and manage the labels of
+the namespace in which it is directly installed into. On some clusters, it's necessary
```

Comment:
Is this relevant to the 2 namespace deployment?

Reply:
Its an attempt at describing why you might need a two namespace config over a three namespace config. Maybe it could be worded better?

Reply:
Perhaps, i couldnt fully understand the reasoning.

Reply:
The problem is multiple sources of truth. Helm wont let you separate the namespace labeling from namespace creation, so have to be done together in the same tool. If some other tool in your Kubernetes cluster needs to create the namespace, then the chart can't in that case.